setup.py

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env python
+
```

Comment:
I do not understand the requirement of the changes in this file.
Can you please elaborate this?

Thanks.

Reply:
i am changing the file because the version of python needs to be checked for networkx as for versions below 3.5 networkx 2.2 is needed so i change the requirements by checking the version in the setup.py file

Reply:
another reason is that the module import.getversion was not working